test/powershell/Scripting/NativeExecution/NativeStreams.Tests.ps1

```diff
@@ -7,13 +7,10 @@ Describe "Native streams behavior with PowerShell" -Tags 'CI' {
 
         $error.Clear()
 
-        $command = [string]::Join('', @(
-            '[Console]::Error.Write(\"foo`n`nbar`n`nbaz\"); ',
-            '[Console]::Error.Write(\"middle\"); ',
-            '[Console]::Error.Write(\"foo`n`nbar`n`nbaz\")'
-        ))
-
-        $out = & $powershell -noprofile -command $command 2>&1
+        # because PowerShell now globs arguments given to native programs,
+        # we use --% so that '::' is not interpreted as a filesystem provider separator;
+        # but this also means that the command cannot be surrounded by quotes as they are not stripped
+        $out = & $powershell 2>&1 -noprofile -command --% [Console]::Error.Write(\"foo`n`nbar`n`nbaz\"); [Console]::Error.Write(\"middle\"); [Console]::Error.Write(\"foo`n`nbar`n`nbaz\")
```

Comment:
The meaning of `::` as file path separator should not anyhow trigger globbing resolving. The `[` makes more sense, I think the explanation comment above needs correction.